pytorch_lightning/plugins/plugin_connector.py

```diff
@@ -94,3 +100,80 @@ def __attach_cluster(self, limit=1):
 
                 # set the cluster
                 self.cloud_environment = plugin
+
+    def _str_to_lightning_custom_plugins(self, plugins: Union[str, list]):
+        """
+        Converts string inputs to corresponding supported lightning plugins.
+        Args:
+            plugins: List of plugins or string to choose lightning plugin.
+
+        Returns: List of plugins where strings are now plugins.
+        """
+        if isinstance(plugins, str):
+            return [self._convert_str_to_plugin(plugins)]
+        return [self._convert_str_to_plugin(plugin) for plugin in plugins]
+
+    def _convert_str_to_plugin(self, plugin):
+        if isinstance(plugin, str):
+            if plugin not in LightningCustomPlugins.__members__:
+                raise MisconfigurationException(
+                    f"{plugin} is not a supported lightning custom plugin. "
+                    f"If you're trying to pass a custom plugin, please pass this as an object to "
+                    f"Trainer(plugins=[MyPlugin()]."
+                    f"Supported plugins as string input: "
+                    f"{(e.name for e in LightningCustomPlugins)}."
```

Comment:
```suggestion
                    f" {plugin} is not a supported lightning custom plugin."
                    " If you're trying to pass a custom plugin, please pass this as an object to"
                    " Trainer(plugins=[MyPlugin()]."
                    f" Supported plugins as string input: {(e.name for e in LightningCustomPlugins)}."
```
let's move spaces at the beginning of the next line and no need for "f" if there is no formatting inside